mypy/checker.py

```diff
@@ -627,6 +627,11 @@ def is_implicit_any(t: Type) -> bool:
                                 if has_any_from_unimported_type(arg_type):
                                     prefix = "Argument {} to \"{}\"".format(idx + 1, fdef.name())
                                     self.msg.unimported_type_becomes_any(prefix, arg_type, fdef)
+                    if ('explicit' in self.options.disallow_any and
```

Comment:
This same piece of code gets repeated almost identically many times. Can you refactor this into a function or a few functions?